themes/default/content/docs/reference/deployments-rest-api/_index.md

```diff
@@ -91,6 +91,7 @@ The executor context defines information about the executor where the Pulumi ope
 
 Image requirements:
 
+* It must be a publicly available image.
```

Comment:
Is this accurate? Like we can't use an ECR repo if there are associated--oh, do we not have anywhere to fill in the creds? The settings have space for them; maybe the UI doesn't...

Reply:
Yeh I don't believe we have anywhere to fill in the creds - does the settings API have accomadation for this? I've never seen it in the code.

Reply:
> does the settings API have accomadation for this? I've never seen it in the code.

Well, I thought it did, but I was wrong. We can accommodate this in the workflow bits but it's not exposed by the deployment settings API. We should fix that.